utils/parse.js

```diff
@@ -1,4 +1,4 @@
-"use strict"
+'use strict'
```

Comment:
please revert all unrelated formatting changes

Reply:
OK, it must be related to my editor `formatOnSave`.

Besides, why the usage of quotes are inconsistent through the project.

Reply:
#1413; however, either way, what eslint doesn't make consistent, shouldn't be autoformatted :-)